app/test_main.py

```diff
@@ -1,3 +1,56 @@
+import pytest
+from typing import Any
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,human_years",
+    [
+        pytest.param(0, 0, [0, 0], id="Mistake during convertation"),
```

Comment:
Use more informative descriptions here: e.g. `0 cat and dog age must equal 0 in human age`